homeassistant/components/waterkotte_heatpump/config_flow.py

```diff
@@ -0,0 +1,106 @@
+"""Config flow for Waterkotte Heatpump integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pywaterkotte import (
+    AuthenticationException,
+    ConnectionException,
+    Ecotouch,
+    EcotouchTags,
+)
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_USERNAME, default="waterkotte"): str,
+        vol.Required(CONF_PASSWORD, default="waterkotte"): str,
```

Comment:
is this the actual default username and password? If so - this is fine, if it is just a placeholder, I would skip this.

Reply:
yes this is he default